src/gui/mzroll/forms/infodialog.ui

```diff
@@ -198,7 +198,15 @@
       </sizepolicy>
      </property>
      <property name="text">
-      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Contact tech support: &lt;a href=&quot;mailto:elmaven@elucidata.io&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;elmaven@elucidata.io&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+      <string>&lt;html&gt;
+&lt;head/&gt;
```

Comment:
@MdAzmatAli could you please fix its indentation?

Reply:
indentation fixed